src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -7,7 +7,7 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// <summary>
 /// The user-interface element that represents the <see cref="MediaElement"/> on Windows.
 /// </summary>
-public class MauiMediaElement : Grid
+public sealed class MauiMediaElement : Grid, IDisposable
```

Comment:
We don't want this to be `sealed`, because we want the users to extend this control if needed